### PR TITLE
chore(browser): render error pages within webview

### LINKED ISSUE #566: Unable to navigate when landing on an error page
## Expected Behavior
When inserting an incorrect URL, I want to use the back arrow to go back.

## Current Behavior
Buttons don't respond

## Steps to Reproduce
1. Go to google.cm
2. Notice error message
3. Click back button
4. You're still on the error page

## Possible Solution
/

## Additional context
I think it should be a great start if we'd fix the issue where you land on an error page and you have to enter a correct URL twice before you're able to navigate.

## Your Environment
* nOS Client version used: latest dev branch
* Environment name and version
    * Node version: any
    * NPM version: any
    * Yarn version: any
* Operating System and version (desktop or mobile):

### PR DESCRIPTION
**Pull request by mhuggins**
## Description
This changes the error page behavior to render within the context of a webview.  This fixes some issues such as back/forward navigation in the process.

## Motivation and Context
Error pages should behave like regular webpages.

## How Has This Been Tested?
Accessing https://invalid.nos.app.

## Screenshots (if appropriate)
error-page-html

## Types of changes
- [x] Chore (tests, refactors, and fixes)
- [ ] New feature (adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** guidelines and confirm that my code follows the code style of this project.
- [ ] Tests for the changes have been added (for bug fixes/features)

#### Documentation
- [ ] Docs need to be added/updated (for bug fixes/features)

## Closing issues
Fixes #566.

### CODE REVIEW

**Comment by DalderupMaurice** [comment]
LGTM - might be sweet if we could center the error message vertically :P

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "bignumber.js": "7.2.1",
     "classnames": "2.2.6",
     "connected-react-router": "4.5.0",
+    "ejs": "^2.6.1",
```
Comment (DalderupMaurice) [inline package.json]: ^

File: static/preloads/handleErrors.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+ejs.fileLoader = (filePath) => {
+  return fs.readFileSync(path.normalize(path.join(__dirname, '..', filePath)));
+};
+
+module.exports = function handleErrors(document) {
+  function renderErrorPage(data) {
+    ejs.renderFile('error.ejs', data, (err, content) => {
+      if (err) {
+        // TODO
```
Comment (DalderupMaurice) [inline static/preloads/handleErrors.js]: TODO?

File: package.json (hunk 1 above)
Comment (DalderupMaurice) [reply package.json]: pushed an update removing this + resolved yarn.lock issue

File: static/preloads/handleErrors.js (hunk 2 above)
Comment (mhuggins) [reply static/preloads/handleErrors.js]: Yeah, I'm not really sure what to do in this case.  Seems like a scenario that we shouldn't ever run into.  Maybe I'll just add a `console.error` call for now.

**Comment by mhuggins** [comment]
@DalderupMaurice I tried vertically centering it initially, but it looked kind of goofy when using the "Details" button since everything jumped.  I left it at the top because of that, but we can always revisit.

File: static/preloads/handleErrors.js (hunk 2 above)
Comment (mhuggins) [reply static/preloads/handleErrors.js]: @DalderupMaurice Updated.

**Pull request merged**